superset/common/request_contexed_based.py

```diff
@@ -16,24 +16,10 @@
 # under the License.
 from __future__ import annotations
 
-from typing import List, TYPE_CHECKING
-
-from flask import g
-
 from superset import conf, security_manager
 
-if TYPE_CHECKING:
-    from flask_appbuilder.security.sqla.models import Role
-
-
-def get_user_roles() -> List[Role]:
```

Comment:
Since we had to change this method, and there were two identical methods in different files, we decided to move it into the security manager where it belongs.